weave-js/src/components/PagePanelComponents/Home/Browse3/filters/common.ts

```diff
@@ -223,8 +226,25 @@ export const FIELD_DESCRIPTIONS: Record<string, string> = {
   'attributes.weave.sys_version': 'Python version used',
 };
 
-export const isWeaveRef = (value: any): boolean => {
-  return typeof value === 'string' && value.startsWith('weave:///');
+// Create a unique symbol for RefString
+const WeaveRefStringSymbol = Symbol('WeaveRefString');
+
+// Define RefString type using the unique symbol
+export type WeaveRefString = string & {[WeaveRefStringSymbol]: never};
+
+/**
+ * `isWeaveRef` is a very conservative check that will ensure the passed
+ * in value is a valid ref string - capabable of being safely parsed into
+ * a Weave ref object. It ensures that the value is a string with the correct
+ * prefix, is parsible, and matches the latest "weave trace" style refs. It
+ * should be used as the appropriate type guard before parsing a ref.
+ */
+export const isWeaveRef = (value: any): value is WeaveRefString => {
```

Comment:
this looks great, do we still need `isRef`? Do we ever have to parse non weave refs?

Reply:
fair - un exported `isRef` and scoped it to a new name reflecting the prefix matching it is doing now